dagshub/data_engine/model/datasource.py

```diff
@@ -120,6 +154,17 @@ def all(self) -> "QueryResult":
         self._check_preprocess()
         return self._source.client.get_datapoints(self)
 
+    def select(self, *selected: Field):
+        """
+        Choose which columns will appear on the query result, what their names will be (alias) and from what time.
+        example:
+        t = int((datetime.datetime.now()-datetime.timedelta(hours=24)).timestamp())
+        q1 = (ds["episode"] > 5).select(Field("episode", as_of_time=t, alias="episode_asof_t"), Field("size"))
+        """
+        self._select = [s.to_dict() for s in selected]
```

Comment:
Looking at the docstring, having to do `Field("size")` looks clunky, I think we should be accepting strings here too:

```suggestion
        self._select = []
        for f in fields:
            if type(f) is str:
                self._select.append(Field(f).to_dict())
            else:
                self._select.append(f.to_dict())
```